src/mudlet.h

```diff
@@ -210,9 +210,9 @@ class mudlet : public QMainWindow, public Ui::MainWindow
     // maps (via signal_profileMapReloadRequested(...))
     void requestProfilesToReloadMaps(QList<QString>);
 
-    const bool getAuditErrorsToConsoleEnabled() { return mIsToDisplayMapAuditErrorsToConsole; }
-    void setAuditErrorsToConsoleEnabled(const bool state) { mIsToDisplayMapAuditErrorsToConsole = state; }
-    void createMapper(bool isToLoadDefaultMapFile = true);
+    const bool getShowMapAuditErrors() { return mshowMapAuditErrors; }
```

Comment:
Actually the `const` specifier on the return value is like nipples on blokes - _of no practical use entirely_ - __unless__ one is returning a reference - which we are not here.  't would be better here to say that we are not modifying the `class` instance with:

    bool getShowMapAuditErrors() const { return mshowMapAuditErrors; }

🤓